engine/crates/graphql-schema-diff/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "graphql-schema-diff"
+version.workspace = true
+edition.workspace = true
+license.workspace = true
+homepage.workspace = true
+keywords.workspace = true
+repository.workspace = true
+
+[dependencies]
+async-graphql-parser = "6"
+async-graphql-value = "6"
```

Comment:
Not using the version from engines because this crate can go to crates.io like the composition crate.